### PR TITLE
Reworking mergify.yml to use a single queue

### LINKED ISSUE #4667: Mergify strict mode depreciation
**Describe the bug**
Mergify will depreciate the 'strict' flag on the 10th Jan, breaking CI

**Product**
CI pipeline

**Version**
N/A

**Additional context**
https://blog.mergify.com/strict-mode-deprecation/

### PR DESCRIPTION
**Pull request by Thingus**
Closes #4667 

### I have:

- [ ] Formatted any Python files with [black](https://github.com/ambv/black)
- [ ] Brought the branch up to date with master
- [ ] Added any relevant Github labels
- [ ] Added tests for any new additions
- [ ] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [ ] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

Rewrites mergify rules to use a default queue.
At present, assuming everything needs to fall into one queue, and the pre-queue checks (previously know as the merge checks) are sufficient to keep the master branch clean.

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+queue_rules:
+  - name: default
+    conditions:
+      - status-success=run_build_pipeline
+      - base=master
```
Comment (jc-harrison) [inline .mergify.yml:5]: I don't think the `base=master` condition is necessary in the queue_rules (but nor is it doing any harm here)

**Review by jc-harrison — COMMENTED** [review]
:+1:

**Comment by Thingus** [comment]
Holding fire til Mergify is happy (the new config file doesn't seem to be propagating, I'm blaming caches)

**Comment by jc-harrison** [comment]
> Holding fire til Mergify is happy (the new config file doesn't seem to be propagating, I'm blaming caches)

I don't think mergify will use the updated config until it's on the main branch

**Pull request merged**